Tests/CoreLocationTests/CLLocationExtensionsTests.swift

```diff
@@ -26,6 +26,18 @@ final class CLLocationExtensionsTests: XCTestCase {
 
         XCTAssertEqual(bearing, 105.619, accuracy: 0.001)
     }
+    
+    @available(macOS 10.12, tvOS 10.0, watchOS 3.0, *)
```

Comment:
Hmm, strange how this is different to last time, but it should match the function.
```suggestion
    @available(iOS 10.12, macOS 10.12, tvOS 10.0, watchOS 3.0, *)
```

Reply:
I got it and push it 😂 It's truly merge time.